tests/resources/mesh_test.go

```diff
@@ -128,7 +129,7 @@ func createAnMTP(t *testing.T, cpName, meshName, mtpName string) {
 	require.NotNil(t, myCp)
 
 	action := shared.ActionAllow
-	resp, err := client.MeshTrafficPermission.PutMeshTrafficPermission(ctx, operations.PutMeshTrafficPermissionRequest{
```

Comment:
We are going back and forth with this `split-put-operations.js`.

1. Initially it was added by @slonka https://github.com/Kong/platform-api/pull/963 as a workaround.
2. Later it was broken and we [released](https://github.com/Kong/terraform-provider-konnect-beta/pull/68) Terraform 0.8.0 without the workaround
3. Now the script is fixed by https://github.com/Kong/platform-api/pull/1763, so you're again introducing the workaround.

According to speakeasy it might be already fixed https://kongstrong.slack.com/archives/C05MLAA216D/p1738254362018029?thread_ts=1738146399.607559&cid=C05MLAA216D. So maybe if we remove `split-put-operations.js` we'll be fine.

Reply:
I'll try and raise a PR on platform-api today if it works.